megalinter/linters/XmlLintLinter.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""
+Use XmlLint to lint xml files
+http://xmlsoft.org/xmllint.html
+"""
+import os
+
+from megalinter import Linter, config
+
+
+class XmlLintLinter(Linter):
+    # Build the CLI command to call to lint a file
+    def build_lint_command(self, file=None):
+        cmd = super().build_lint_command(file)
+
+        if (
+            self.apply_fixes is True
+            and self.cli_lint_fix_arg_name is not None
+            and config.get("XML_XMLLINT_AUTOFORMAT", "false") == "true"
+        ):
+            if self.cli_lint_mode == "file":
+                os.environ["XMLLINT_INDENT"] = config.get("XML_XMLLINT_INDENT", "  ")
```

Comment:
@nvuillam this is correct, isn't it? As I already told you, the fact of how many spaces it applies is done through an environment variable

Reply:
why defining indent only if cli_lint_mode = file ? not for all modes (including list_of_files) ?

Reply:
oh ok I see the code ^^ autoformat works only with one file as parameter ??